region/maridia/outer/Mt. Everest.json

```diff
@@ -3162,6 +3162,33 @@
         "Then Gravity jump two or more times to chain it up to the door."
       ]
     },
+    {
+      "link": [5, 5],
+      "name": "Sciser Ice Clip Door Lock Skip",
+      "requires": [
+        "Gravity",
+        {"or": [
+          {"and": [
+            "canGravityJump",
+            "canBePatient"
+          ]},
+          "SpaceJump"
+        ]},
+        {"ammo": {"type": "Super", "count": 1}},
+        "h_canXRayMorphIceClip"
+      ],
+      "bypassesDoorShell": true,
+      "flashSuitChecked": true,
+      "note": [
+        "Lure a crab to the top door.",
+        "Use a Super to knock it off and freeze it below the door.",
+        "Use Morph and X-Ray to stand up, then jump to clip past the door shell."
+      ],
+      "devNote": [
+        "This doesn't seem possible to get the crab frozen at the high pixel position.",
+        "In fact, it doesn't seem possible to do at all without Morph and X-Ray."
+      ]
```

Comment:
Ive been testing ice clip doorlock skips and found that this is only possible with X-Ray+Morph (precise clip and high pixel clip are not possible). The freeze is frame perfect and requires Gravity or HiJump to clip through.